### DISCUSSION TITLE
dependencies vs devDependencies

### DISCUSSION THREAD

**Discussion by P4sca1** [nuxt/nuxt#16975] [Questions]
Since nuxt 3 is now a build-only tool, I'm wondering whether I should install nuxt modules, external composables, my ui library, etc to `dependencies` or `devDependencies`?

If I understand correctly, everything can be installed into `devDependencies`, because nitro will create a `package.json` and `node_modules` in `.output` after building, which will contain only the dependencies that are required during runtime.
Is that correct?

**Comment by danielroe** [comment]
Yes, that is right 👍
